shanoir-ng-datasets/src/main/java/org/shanoir/ng/examination/service/ExaminationServiceImpl.java

```diff
@@ -146,6 +154,9 @@ public void deleteById(final Long id, ShanoirEvent event) throws ShanoirExceptio
 					eventService.publishEvent(event);
 				}
 			}
+			String seriesInstanceUID = studyInstanceUIDHandler.findStudyInstanceUID(examination);
```

Comment:
Hi @pierrehenri-dauvergne, thank you for your PR.
A minor thing, but please correct. Rename to studyInstanceUID,
because, that is what you are getting here from sIUIDHandler.
Cheers, Michael